cylc/flow/cfgspec/globalcfg.py

```diff
@@ -409,7 +409,7 @@
             Examples::
 
                atom --wait
-               code -nw
+               code -new-window --wait
```

Comment:
The `new-window` needs `--` not `-`, what's wrong with the short options?

Reply:
IMO the long versions don't require as many people to go and look them up

Reply:
Please fix the new-window opt.

Reply:
```suggestion
               code --new-window --wait
```